src/core_plugins/kibana/inject_vars.js

```diff
@@ -10,7 +10,6 @@ export function injectVars(server) {
   const regionmapsConfig = serverConfig.get('regionmap');
   const mapConfig = serverConfig.get('map');
 
-
```

Comment:
ups ?

Reply:
?

Reply:
the only change in this file is the removal of this empty line ....

Reply:
oh, oops! ;)